Library/PagedContainer/PagedContainerViewController.swift

```diff
@@ -0,0 +1,138 @@
+import Combine
+import Foundation
+import UIKit
+
+open class PagedContainerViewController: UIViewController {
+  private weak var activeController: UIViewController? = nil
+  private var subscriptions = Set<AnyCancellable>()
+  private let viewModel = PagedContainerViewModel()
```

Comment:
Philosophically, I put everything that seemed like `UIKit` logic in the `ViewController`, and everything that seemed like "how it's displayed" logic in the view model. However, that ended up meaning the view model was _very small_ since it's a pretty simple concept for a page 🤷‍♀️